test/integration/targets/uri/tasks/main.yml

```diff
@@ -495,4 +495,7 @@
   include_tasks: redirect-urllib2.yml
 
 - name: Test follow_redirects=all
-  include_tasks: redirect-all.yml
\ No newline at end of file
+  include_tasks: redirect-all.yml
```

Comment:
Use `import_tasks` since there's not need for dynamic inclusion.

Reply:
No. Static imports are more straight forward and preferred unless dynamic inclusion is required, which is not the case here.